asteroid/metrics.py

```diff
@@ -101,3 +101,40 @@ def get_metrics(
         return average_arrays_in_dic(utt_metrics)
     else:
         return utt_metrics
+
+
+def batch_metrics(inputs, targets, estimates, sizes,
+                  sample_rate=16000, metrics_list="all",
+                  average=True, compute_permutation=False,
+                  ignore_metrics_errors=False, filename=None):
```

Comment:
@jonashaag I realize that `filename` is used nowhere no..

Reply:
It is passed to `get_metrics` below for better error reporting

Reply:
Oh, I guess I completely misunderstood the point. I thought the filename was supposed to be a log file.. 
Now I see.